src/hooks/Transactor.ts

```diff
@@ -196,7 +196,7 @@ export function handleTransactionException({
   txOpts?: TxOpts
   missingParam?: string
   functionName: string
-  cv: CV | undefined
+  cv: '1' | CV2V3 | undefined // TODO should this be CV?
```

Comment:
**question** should we just revert this line?

Reply:
oops this was corrected but should have removed the TODO